recipes/rtk/meta.yaml

```diff
@@ -23,6 +23,7 @@ test:
   files:
     - table.tsv
   commands:
+    - rtk -h
```

Comment:
I think this test is skipped if there is a run_test.sh

Reply:
According to here (https://bioconda.github.io/contributor/linting.html#missing-tests) the two alternatives are “and/or” so I assumed they were not mutually exclusive. 
From the automatic test I also had feedback that the command was not ignored because of the presence of the bash script.

Reply:
The command is used in the mulled test and the shell script outside of it.